src/main/scala/GuessWhoGame/Board.scala

```diff
@@ -1,7 +1,116 @@
 package GuessWhoGame
 
-case class Board (Characters: List[Character]) {
+import scala.util.Random
+
+//values could be more domain specific, name/selecting could be within character case class, then able to test separately
+case class Board(characters: List[Character]) {
   def printCharacterNames(): Unit = {
-    Characters.foreach(Character => println(Character.name))
+    characters.foreach(Character => println(Character.name))
+  }
+
+  val selectedCharacter: Character = selectRandomCharacter(characters)
+  var remainingCharacters: List[Character] = characters
+
```

Comment:
Scala favours immutability. Can we eliminate vars?

Reply:
Vars have been eliminated with new variations produced each time.